proto/panacea/did/v2/query.proto

```diff
@@ -11,8 +11,8 @@ option go_package = "github.com/medibloc/panacea-core/x/did/types";
 
 // Query defines the gRPC querier service.
 service Query {
-	// DIDDocumentWithSeq returns a DID Document with a sequence number.
-	rpc DIDDocumentWithSeq(QueryDIDRequest) returns (QueryDIDResponse) {
+	// DID returns a DID Document with a sequence number.
+	rpc DID(QueryDIDRequest) returns (QueryDIDResponse) {
```

Comment:
Due to 
> {"path":"proto/panacea/did/v2/query.proto","start_line":15,"start_column":58,"end_line":15,"end_column":74,"type":"RPC_RESPONSE_STANDARD_NAME","message":"RPC response type \"QueryDIDResponse\" should be named \"DIDDocumentWithSeqResponse\" or \"QueryDIDDocumentWithSeqResponse\"."}

Similar as https://github.com/medibloc/panacea-core/blob/d1fbdd948e963c587407d9c865514e71bbea7e7b/proto/panacea/aol/v2/query.proto#L17